frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/useProjectActionsForm.ts

```diff
@@ -1,14 +1,21 @@
 import { useActions } from 'hooks/api/getters/useActions/useActions';
-import { IAction, IActionSet } from 'interfaces/action';
+import { IActionSet } from 'interfaces/action';
 import { useEffect, useState } from 'react';
+import { UIAction } from './ActionItem';
 
-enum ErrorField {
+export enum ErrorField {
```

Comment:
Are we importing this anywhere?

Reply:
I thought this would be needed in some components later, but I didn't get to work on errors